### PR TITLE
Be more consistent about how stack traces are output from flutter_test

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
I need to add a test.

File: packages/flutter_test/lib/src/binding.dart [hunk 1]
```diff
@@ -1021,3 +1022,12 @@ class _EmptyStack implements StackTrace {
   @override
   String toString() => '';
 }
+
+StackTrace _unmangle(StackTrace stack) {
+  if (stack is stack_trace.Trace)
+    return stack.vmTrace;
+  if (stack is stack_trace.Chain)
+    return stack.toTrace().vmTrace;
+  print('got: ${stack.runtimeType}');
```
Comment (Hixie) [inline packages/flutter_test/lib/src/binding.dart]: and i need to remove this line...

File: packages/flutter_test/lib/src/binding.dart [hunk 2]
```diff
@@ -1021,3 +1022,11 @@ class _EmptyStack implements StackTrace {
...
   String toString() => '';
 }
+
+StackTrace _unmangle(StackTrace stack) {
+  if (stack is stack_trace.Trace)
+    return stack.vmTrace;
+  if (stack is stack_trace.Chain)
+    return stack.toTrace().vmTrace;
+  return stack;
+}
```
Comment (jason-simmons) [inline packages/flutter_test/lib/src/binding.dart]: add a newline at the end of the file

**Comment by jason-simmons** [comment]
LGTM

File: packages/flutter_test/lib/src/binding.dart (hunk 2 above)
Comment (Hixie) [inline packages/flutter_test/lib/src/binding.dart]: roger wilco

**Comment by Hixie** [comment]
will check in on green

**Pull request merged**